pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -271,9 +275,13 @@ public void recordLatency(EventType eventType, long latencyMs) {
                 pulsarStats.recordZkLatencyTimeValue(eventType, latencyMs);
             }
         };
+
+        this.delayedDeliveryTrackerFactory = DelayedDeliveryTrackerLoader
+                .loadDelayedDeliveryTrackerFactory(pulsar.getConfiguration());
     }
 
     public void start() throws Exception {
+        this.delayedDeliveryTrackerFactory.initialize(pulsar.getConfiguration());
```

Comment:
isn't `initialize` already called when the factory is loaded? this is a double-initialize action.

Reply:
Yes, I moved the initialize on the start() but that's not consistent with other classes. I'll change it back to call initialize immediately after the class is instantiated.

Reply:
This was part of the previous mistake. I had it removed from load manager thinking I had it removed from the tracker loader..